### PR TITLE
Update editor header design

### PR DESCRIPTION
**Pull request by felippenardi**
#### What problem is this solving?
We've updated the sidebar block editor header. This is part of a series
of UX improviments that are coming to the Site Editor.

#### How should this be manually tested?

- Edit a block
- See the new header

[Workspace](https://ch25056--storecomponents.myvtex.com/admin/cms/site-editor)

#### Screenshots or example usage

Before:
Screen Shot 2019-10-31 at 17 22 09

After:
Screen Shot 2019-10-31 at 17 21 51

#### Type of changes

| ✔️  | Type of Change                                                                            |
| --- | ----------------------------------------------------------------------------------------- |
| \_  | Bug fix                                |
| \_  | New feature                        |
| \_  | Breaking change  |
| \_  | Technical improvements  |

### CODE REVIEW

File: react/components/EditorContainer/Sidebar/BlockEditor/EditorHeader.tsx [hunk 1]
```diff
@@ -32,60 +30,59 @@ const EditorHeader: React.FC<Props> = ({
...
-              role="button"
-              tabIndex={0}
-            >
-              <IconArrowBack />
-            </span>
+            <React.Fragment>
+              <span className="outline-0">
+                <IconArrowBack />
+              </span>
+              <span className="ml3 fw5 t-mini">BACK</span>
```
Comment (alcar) [inline react/components/EditorContainer/Sidebar/BlockEditor/EditorHeader.tsx]: Missing i18n.

File: react/components/EditorContainer/Sidebar/BlockEditor/EditorHeader.tsx [hunk 2]
```diff
@@ -32,60 +30,59 @@ const EditorHeader: React.FC<Props> = ({
...
+        <div>
           {onListClose && (
             <span
               className="pointer outline-0 c-on-base hover-c-action-primary"
-              onClick={onListClose}
-              onKeyDown={handleCloseListKeyDown}
               role="button"
               tabIndex={0}
+              onClick={onListClose}
+              onKeyDown={handleCloseListKeyDown}
```
Comment (alcar) [inline react/components/EditorContainer/Sidebar/BlockEditor/EditorHeader.tsx]: :pick: Why did you reorder those?

File: react/components/EditorContainer/Sidebar/BlockEditor/EditorHeader.tsx [hunk 3]
```diff
@@ -32,60 +30,59 @@ const EditorHeader: React.FC<Props> = ({
...
-                defaultMessage: 'Content list',
+            <span
+              className="pointer outline-0 c-action-primary hover-c-action-primary ttu f7 fw5"
+              role="button"
+              tabIndex={0}
+              onClick={onListOpen}
+              onKeyDown={handleOpenListKeyDown}
+            >
+              {intl.formatMessage({
+                defaultMessage: 'Configuration',
```
Comment (alcar) [inline react/components/EditorContainer/Sidebar/BlockEditor/EditorHeader.tsx]: The default value for this key is "Configurations".

**Comment by pedroalexandria** [comment]
Where we should have BACK, we have Configurations:

image

Also, I think the arrow is a little bit higher than layout. Please, check the screenshot from Figma below:

image

**Comment by pedroalexandria** [comment]
Please, be sure the navigation buttons are aligned. They must be within the red lines.

image

**Comment by felippenardi** [comment]
@pedroalexandria Just fixed it! Can you please check again?
Screen Shot 2019-11-04 at 16 31 01

**Comment by pedroalexandria** [comment]
@felippenardi That's all right :)

**Comment by thiagomurakami** [comment]
Maybe unrelated to this PR, but should we change all `content` to `configuration`?

When you click `Configurations`, should the button text be `New configuration` instead of `New content`?
There's also the `Untitled content` default string

**Comment by felippenardi** [comment]
`Untitled content` still remains for now, so far this is the design decision we are choosing.

Let's address the terminology questions in another PR.

**Pull request merged**